channelserver/src/settings.rs

```diff
@@ -0,0 +1,39 @@
+use std::env;
+
+use config::{Config, ConfigError, Environment, File};
+
+static PREFIX: &str = "PAIR";
+
+#[derive(Debug, Deserialize)]
+pub struct Settings {
+    pub hostname: String,  // server hostname (localhost)
+    pub port: u16,         // server port (18000)
+    pub max_clients: u8,   // Max clients per channel 2
+    pub timeout: u64,      // seconds before channel timeout (300)
+    pub max_exchanges: u8, // Max number of messages before channel shutdown (8)
+    pub max_data: u64,     // Max amount of data octets to exchange (0 ; unlimited)
+    pub debug: bool,       // In debug mode?
+    pub verbose: bool,     // Verbose Errors?
+}
+
+impl Settings {
+    pub fn new() -> Result<Self, ConfigError> {
+        let mut settings = Config::default();
+
+        settings.set_default("debug", false)?;
+        settings.set_default("verbose", false)?;
+        settings.set_default("max_exchanges", 0)?;
+        settings.set_default("timeout", 300)?;
+        settings.set_default("max_clients", 2)?;
```

Comment:
Ah, OK, we can limit the channels to 2 in the config, but it looks like this is not implemented in the code yet.